src/CommunityToolkit.Maui.UnitTests/Alerts/Toast_Tests.cs

```diff
@@ -0,0 +1,73 @@
+﻿using CommunityToolkit.Maui.Alerts;
+using CommunityToolkit.Maui.Core;
+using FluentAssertions;
+using Xunit;
+
+namespace CommunityToolkit.Maui.UnitTests.Alerts;
+
+public class Toast_Tests : BaseTest
+{
+	IToast toast = new Toast();
```

Comment:
Should we make this readonly?

```suggestion
	readonly IToast toast = new Toast();
```